packages/adapter/src/request-building/cookies.ts

```diff
@@ -12,7 +12,7 @@ export function buildHeader(cookies: Parameters): CookieHeader | object {
 
   const keys = Object.keys(cookies);
 
-  const cookieValues = keys.reduce((values, key) => {
+  const cookieValues = keys.reduce((values: string[], key) => {
```

Comment:
Nit: I'd type the initial accumulator instead: `[] as string[]` - rest gets infered